docker-compose.yaml

```diff
@@ -160,25 +158,6 @@ services:
     ports:
       - 80:80
 
-  code-server:
```

Comment:
We no longer need this?

Reply:
It's the `bandsv` thing. No longer needed.